lib/fog/vcloud_director/generators/compute/compose_common.rb

```diff
@@ -34,12 +39,17 @@ def has_source_items?
 
           def build_vapp_instantiation_params(xml)
             xml.Description @configuration[:Description] if @configuration[:Description]
-            
-            vapp = @configuration[:InstantiationParams]
-            if vapp 
-              xml.InstantiationParams {
+            xml.InstantiationParams do
```

Comment:
Previously `InstantiationParams` were only added when it was also available in the `@configuration`. Otherwise, there was no such element in the submitted XML.

Will this cause problems if we submit an empty element?